src/App.tsx

```diff
@@ -1,24 +1,224 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
```

Comment:
Good Job!
Here are the next problems: 
1) Toggle button does not work.
2) If there is no completed todos 'comlete all button' should desapear